.chloggen/generic-configunmarshaler.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
It is not breaking, it is an internal package.

Reply:
Fixed